5_databases/week2/More_queries.sql

```diff
@@ -0,0 +1,28 @@
+SELECT * FROM user;
+
+SELECT user.name, user.email, task.title 
+FROM user JOIN user_task on user.id = user_id
+          JOIN task ON task.id= task_id
+WHERE email LIKE '%@spotify.com%';
+
+
+
+SELECT user.name, task.title, status.name 
```

Comment:
Something is not right here. I get tasks back that are also Done and In Progress. We only want the Not started ones.

Try again.